document_merge_service/api/unoconv.py

```diff
@@ -9,15 +9,39 @@
 
 
 class Unoconv:
-    def __init__(self, pythonpath, unoconvpath):
+    def __init__(self, pythonpath, unoconvpath, server=None, port=2002):
+        """
+        Convert documents with unoconv command-line utility.
+
+        :param filename: str() - path to the python interpreter
+        :param convert: str() - path to the unoconv binary
+        :param connection: str() - connection string to an unoconv listener
+        """
         self.pythonpath = pythonpath
         self.unoconvpath = unoconvpath
+        self.server = server
+        self.port = port
+
+    @property
+    def cmd(self):
+        cmd = [self.pythonpath, self.unoconvpath]
+
+        if self.server:  # pragma: no cover
```

Comment:
@open-dynaMIX Should we test this?

Reply:
If it's possible without too much trouble, it would be great!

Reply:
I've added a test.

Reply:
:+1: